RELEASES.md

```diff
@@ -0,0 +1,37 @@
+# Version 1.2
+
+### UI Improvements
+
+A lot of UI improvements
+
+![](./docs/interface-v1.2.gif)
+
+### New features
```

Comment:
I would put it above the UI.

You can also go for a top 5 chnages, and then a detailed list splitted to categories )new features, buf fixes, performance, ui, ...)